elements/pfe-jump-links/src/pfe-jump-links-nav.html

```diff
@@ -26,17 +26,17 @@ <h2 class="sr-only" hidden>Page navigation</h2>
     <nav>
       <h2 class="sr-only" hidden>Page navigation</h2>
       ${ this.has_slot("heading") ?
-      `<slot class="heading" name="heading" id="heading"></slot>`
-      : `<h4 class="heading" id="heading">Jump to section</h4>`
+      `<slot class="pfe-jump-links-nav--heading" name="pfe-jump-links-nav--heading" id="pfe-jump-links-nav--heading"></slot>`
```

Comment:
```suggestion
      `<slot class="pfe-jump-links__heading" name="pfe-jump-links--heading"></slot>`
```

Reply:
Classes use BEM naming so the region is `__` underscore prefixed.  Slots use a more straight forward naming so these are just `--` prefixed for an easier developer experience.

Reply:
Also, since the component is called `pfe-jump-links`, I suggest using that as your base instead of adding `-nav` which might lead to some confusion.